addons/microsoft_calendar/security/ir.model.access.csv

```diff
@@ -1,2 +1,3 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 microsoft_calendar_account_reset,microsoft_calendar_account_reset_access_right,model_microsoft_calendar_account_reset,base.group_system,1,1,1,0
+microsoft_calendar.access_microsoft_calendar_credentials,access_microsoft_calendar_credentials,microsoft_calendar.model_microsoft_calendar_credentials,base.group_user,1,1,0,0

```

Comment:
You need security xml file to avoid giving the access to read and write on everybody's token to all users.
You can look at google_calendar_security.xml.

Reply:
@arj-odoo good catch, I read too fast the PR!